calcom/cal.com
# fix: small UI improvements for troubleshooter

- [x] fixed padding
- [x] i18n fixes
- [x] show troubleshooter on your own page
- [x] bug: when visiting: http://localhost:3000/availability/troubleshoot?eventType=paid it doesn't select "paid" and instead the first entry
![CleanShot 2023-11-24 at 23 12 44@2x](https://github.com/calcom/cal.com/assets/8019099/2a74e53e-f7db-4f3b-b056-a0ef366fd0bb)


after:

![CleanShot 2023-11-24 at 23 03 14@2x](https://github.com/calcom/cal.com/assets/8019099/fa48004b-bd37-48b2-9def-7d21def59593)


![CleanShot 2023-11-24 at 23 02 45@2x](https://github.com/calcom/cal.com/assets/8019099/2e8eb4d7-5e78-4212-b7dc-cbf299782207)

## how to test
- visit your own link on localhost
- see "need help" button
- click it

- visit someone else's link on local host
- don't see the "need help" button
